addons/point_of_sale/static/src/js/models.js

```diff
@@ -202,6 +202,7 @@ exports.PosModel = Backbone.Model.extend({
         fields: ['name','street','city','state_id','country_id','vat','lang',
                  'phone','zip','mobile','email','barcode','write_date',
                  'property_account_position_id','property_product_pricelist'],
+        domain: function(self) { return [['company_id', 'in', [self.company && self.company.id, false]]] },
```

Comment:
@mgh-odoo Could you modify the domain by ['company_id', '?=', self.company.id] and test that it works well.

Thanks

Reply:
@pimodoo it will not work in our case because we have some customers 'False' on the left side.

Reply:
@mgh-odoo yes, the operator '=?' takes the value or False

Reply:
@pimodoo reverted to the previous `['company_id', 'in', [self.company && self.company.id, false]` domain. Thanks!